sklearnex/basic_statistics/incremental_basic_statistics.py

```diff
@@ -53,40 +55,76 @@ class IncrementalBasicStatistics(BaseEstimator):
     batch_size : int, default=None
         The number of samples to use for each batch. Only used when calling
         ``fit``. If ``batch_size`` is ``None``, then ``batch_size``
-        is inferred from the data and set to ``5 * n_features``, to provide a
-        balance between approximation accuracy and memory consumption.
+        is inferred from the data and set to ``5 * n_features``.
 
-    Attributes (are existing only if corresponding result option exists)
+    Attributes
     ----------
-        min : ndarray of shape (n_features,)
+        min_ : ndarray of shape (n_features,)
```

Comment:
Why the attribute name is changed?

Reply:
https://github.com/intel/scikit-learn-intelex/pull/2038